docs/reference/commandline/cli.md

```diff
@@ -265,6 +273,15 @@ Following is a sample `config.json` file:
 {% endraw %}
 ```
 
+### Experimental features
+
+To enable [experimental features](https://success.docker.com/article/experimental-features),
+edit the `config.json` file and set `experimental` to `enabled`.
+
+You can also enable experimental features from the Docker Desktop menu. Click
```

Comment:
Since this documentation is specifically for the command-line cli, should not have desktop docs here. Experimental feature enablement is already covered in docker desktop docs https://github.com/docker/docker.github.io/blob/master/docker-for-mac/index.md#experimental-features

Reply:
Agreed; we could link to it though